src/features/controls.ts

```diff
@@ -24,6 +24,31 @@ import {IdentNode, NumberNode, numberNode, parseExpressions} from '../styles/par
 import {DEFAULT_FOV_DEG} from '../three-components/Model.js';
 import {ChangeEvent, ChangeSource, SmoothControls} from '../three-components/SmoothControls.js';
 import {Constructor} from '../utilities.js';
+import {timeline} from '../utilities/animation.js';
+
+// NOTE(cdata): The following "animation" timing functions are deliberately
+// being used in favor of CSS animations. In Safari 12.1 and 13, CSS animations
+// would cause the interaction prompt to glitch unexpectedly
+// @see https://github.com/GoogleWebComponents/model-viewer/issues/839
+const PROMPT_ANIMATION_TIME = 5000;
+
+const wiggle = timeline(0, [
+  {frames: 6, value: 0},
```

Comment:
I know you explain this down in animation.ts, but it might be nice to have a short overview comment here. I guess the frames are portions of the overall animation time, so more like frame lengths than frame times, yeah?

Reply:
Will fix